zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -441,7 +440,7 @@ pub fn create_proposed_transaction<DbT, ParamsT, InputsErrT, FeeRuleT>(
     min_confirmations: NonZeroU32,
     change_memo: Option<MemoBytes>,
 ) -> Result<
-    DbT::TxRef,
+    TxId,
```

Comment:
[Non - Blocking] Although small, this is a great improvement!

I'm mostly sure it will break the ffi layer so it will be good to track this hiccup in the ffi repository on iOS and the android sdk r